### ISSUE TITLE
Training accuracy increases, val accuracy stays the same

### ISSUE THREAD

**Issue by WassimOrabi** [keras-team/keras#13136]
I am trying to train a CNN using frames that portray me shooting a ball through a basket. And my aim is for the network to be able to classify the result( hit or miss) correctly. When I train the network, the training accuracy increases slowly until it reaches 100%, while the validation accuracy remains around 65% (It is important to mention here that 65% is the percentage of shots that have a Miss label)   Does anyone have experience with a similar problem?
I am using PyTorch and Resnet18 ( have tried other architectures as well but they all gave the same result). My frames are jpg images of sie 224.
As an optimizer, both Adam and SGD gave the same result 
Thank you in advance

**Comment by gowtham-kp** [comment]
@Can you please share a code snippet as the problem description is too vague to jump into any conclusions. Also you can refer to a [similar issue](https://datascience.stackexchange.com/questions/13607/validation-loss-and-accuracy-remain-constant) here and let me know if it helps. Thanks!
